examples/storm-perf/pom.xml

```diff
@@ -22,8 +22,8 @@
     <parent>
         <artifactId>storm</artifactId>
         <groupId>org.apache.storm</groupId>
-        <version>1.1.1-SNAPSHOT</version>
-        <relativePath>..</relativePath>
+        <version>1.1.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
```

Comment:
@ptgoetz 
Fixing this to `../../pom.xml` resolved the error on `mvn clean install -DskipTests`. Please address this.